parent/build.sh

```diff
@@ -34,12 +36,27 @@ docker build \
 
 cd ../quoteserver
 docker build \
+--add-host quoteserve.seng:172.20.0.1 \
```

Comment:
Probably don't need this line if you are passing it as build-arg below.

Reply:
You're right lol